src/Map.tsx

```diff
@@ -10,12 +10,14 @@ import useEvents from './utils/useEvents'
 export interface MapProps {
   events?: LeafletEventHandlerFnMap
   options?: MapOptions
+  setInstance?: (instance: L.Map) => void
```

Comment:
Instead of passing `setInstance` would it not be better to pass `mapInstance` as a prop instead?

Reply:
I wouldn't do that, the Map component should be the only responsible for creating a L.Map object